sdks/python/tests/e2e/test_experiment.py

```diff
@@ -77,7 +77,147 @@ def task(item: Dict[str, Any]):
         experiment_metadata={"model_name": "gpt-3.5"},
         traces_amount=3,  # one trace per dataset item
         feedback_scores_amount=1,
-        prompt=prompt,
+        prompts=[prompt],
+    )
+
+    retrieved_experiment = opik_client.get_experiment_by_name(experiment_name)
+    experiment_items_contents = retrieved_experiment.get_items()
+    assert len(experiment_items_contents) == 3
+
+    EXPECTED_EXPERIMENT_ITEMS_CONTENT = [
+        experiment_item.ExperimentItemContent(
+            id=ANY_BUT_NONE,
+            dataset_item_id=ANY_BUT_NONE,
+            trace_id=ANY_BUT_NONE,
+            dataset_item_data={
+                "input": {"question": "What is the of capital of France?"},
+                "expected_model_output": {"output": "Paris"},
+            },
+            evaluation_task_output={"output": "Paris"},
+            feedback_scores=[
+                {
+                    "category_name": None,
+                    "name": "equals_metric",
+                    "reason": None,
+                    "value": 1.0,
+                }
+            ],
+        ),
+        experiment_item.ExperimentItemContent(
+            id=ANY_BUT_NONE,
+            dataset_item_id=ANY_BUT_NONE,
+            trace_id=ANY_BUT_NONE,
+            dataset_item_data={
+                "input": {"question": "What is the of capital of Germany?"},
+                "expected_model_output": {"output": "Berlin"},
+            },
+            evaluation_task_output={"output": "Berlin"},
+            feedback_scores=[
+                {
+                    "category_name": None,
+                    "name": "equals_metric",
+                    "reason": None,
+                    "value": 1.0,
+                }
+            ],
+        ),
+        experiment_item.ExperimentItemContent(
+            id=ANY_BUT_NONE,
+            dataset_item_id=ANY_BUT_NONE,
+            trace_id=ANY_BUT_NONE,
+            dataset_item_data={
+                "input": {"question": "What is the of capital of Poland?"},
+                "expected_model_output": {"output": "Warsaw"},
+            },
+            evaluation_task_output={"output": "Krakow"},
+            feedback_scores=[
+                {
+                    "category_name": None,
+                    "name": "equals_metric",
+                    "reason": None,
+                    "value": 0.0,
+                }
+            ],
+        ),
+    ]
+    assert_equal(
+        sorted(
+            EXPECTED_EXPERIMENT_ITEMS_CONTENT,
+            key=lambda item: str(item.dataset_item_data),
+        ),
+        sorted(experiment_items_contents, key=lambda item: str(item.dataset_item_data)),
+    )
+
+
+def test_experiment_creation_via_evaluate_function__prompts_arg_used__happyflow(
+    opik_client: opik.Opik, dataset_name: str, experiment_name: str
+):
+    # TODO: this test is not finished, it does not check experiment items content
```

Comment:
No need to duplicate this comment, and it is actually outdates since we check the experiment items content already :)